packages/elysia/src/ElysiaAdapter.ts

```diff
@@ -136,7 +158,7 @@ export class ElysiaAdapter implements IServerAdapter {
       const relativePath = path.substring(path.indexOf('dist') + 4).replaceAll('\\', '/');
       this.plugin.get(relativePath, async () => {
         const fileContent = await fsPromises.readFile(path);
-        return new Response(fileContent, {
+        return new Response(fileContent.toString(), {
```

Comment:
How does this related to the "onParse" part?

Reply:
Why your issues are not reproduces in the `with-elysia` example?

Reply:
Yeah, but there was a PR that made useage of this Elysia adapter sutable in Node as well...